src/QMCDrivers/VMC/VMC.cpp

```diff
@@ -49,6 +50,12 @@ VMC::VMC(MCWalkerConfiguration& w,
   m_param.add(UseDrift, "useDrift", "string");
   m_param.add(UseDrift, "usedrift", "string");
   m_param.add(UseDrift, "use_drift", "string");
+  m_param.add(SpinMoves, "spinMoves", "string");
+  m_param.add(SpinMoves, "spinmoves", "string");
+  m_param.add(SpinMoves, "spin_moves", "string");
```

Comment:
We will need to set official input tag naming rules. Only pick one and error others as unidentified input. Not a problem of this PR, we will settle that latter when deleting drivers.

Reply:
Full agreement on this one. Please pick a single tag.

Reply:
fixed